src/lib/matcher.rs

```diff
@@ -45,6 +45,7 @@ fn bases_colored(
     let mut colored_quals = Vec::with_capacity(bases.len());
 
     // Merge the ranges into a bit mask, with 1 indicating that base is part of a pattern match
+    println!("{:?}", &bases.len());
```

Comment:
```suggestion
```

Reply:
@samfulcrum please remove this `println!`